lib/Models/Cesium.js

```diff
@@ -423,6 +423,138 @@ Cesium.prototype.computePositionOnScreen = function(position, result) {
     }
 };
 
+/**
+ * Updates the order of layers on the Leaflet map to match the order in the Now Viewing pane.
+ */
+Cesium.prototype.updateLayerOrder = function() {
+    // move alwaysOnTop layers to the top
+    var items = this.terria.nowViewing.items;
+    var scene = this.scene;
+    for (var l = items.length - 1; l >= 0; l-- ) {
+        if (items[l].imageryLayer && items[l].keepOnTop) {
+            scene.imageryLayers.raiseToTop(items[l].imageryLayer);
+        }
+    }
+};
+
+Cesium.prototype.updateLayerOrderAfterReorder = function() {
+    // because this Cesium model does the reordering via raise and lower, no action needed.
+};
+
+// useful for counting the number of items in composite and non-composite items
+function numSubItems(item) {
```

Comment:
Another total nitpick: avoid abbreviations, and name functions with verbs.  `countNumberOfSubItems` maybe.